followthemoney_compare/lib/compare2_tk.py

```diff
@@ -0,0 +1,155 @@
+import itertools
+import math
+import warnings
+
+from fuzzywuzzy import fuzz
+from normality import normalize
+import fingerprints
+from followthemoney.exc import InvalidData
+from followthemoney.types import registry
+
+
+# Compare weights come from the glm-bernouli model in followthemoney-predict
+# COMPARE_WEIGHTS = {
+#     registry.name: 12.275729155073371,
+#     registry.country: 1.0494517476987815,
+#     registry.date: 6.960245940274218,
+#     registry.identifier: 5.2209896558064175,
+#     registry.address: 6.456137299747168,
+#     registry.phone: 3.538892687331418,
+#     registry.email: 14.115925628770384,
+#     registry.iban: 0.019140301711998726,
+#     registry.url: 3.211995327345834,
+#     None: -11.91521189545115,
+# }
+COMPARE_WEIGHTS = {
```

Comment:
Grr... I wrote a long comment here but somehow it got lost. Essentially, I don't think we should recreate `followthemoney.compare` here because the two codebases could diverge. Instead, I think we should keep the new `COMPARE_WEIGHTS` (and name them `COMPARE_WEIGHTS_RANDOM` or something so there is no confusion as to what they do) and use followthemoney's `compare.compare_scores` and `compare._compare` to create the scores.

That would look like:

```
from followthemoney import compare
scores = compare.compare_scores(model, entitty1, entity2)
result_ftm = compare._compare(scores, compare.COMPARE_WEIGHTS)
result_ours = compare._compare(scores, COMPARE_WEIGHTS_RANDOM)
```

That being said, ideally I think _this repo_ should be the arbiter of what comparisons are.  Instead of importing from followthemoney, we should probably stick to using our own model objects defined in `followthemoney_compare.model`. You can see the [model_base.py](https://github.com/alephdata/followthemoney-compare/blob/main/followthemoney_compare/models/model_base.py) object (which all models inherit from) already defines some utility evaluation methods.

Reply:
> Could you take the code in your evaluate.py and encode it as a model method

This makes sense. Should make it with n models rather than just two?

I'll change how the weights are used as well. I think it might be good to use this PR just to add the evaluation bits. Still not a 100% sure / clear what's the best way to play around with different comparisons, but I'll ask those questions once this is done.